### PR TITLE
SQL Changes

### PR DESCRIPTION
**Pull request by dwppalmer95**

### CODE REVIEW

File: client/src/pages/UserProfile.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export function UserProfile(props) {
 
   useEffect(() => {
     if (userData._id) {
-      fetch(`/reviews/${userData._id}`)
+      fetch('/reviews/${userData._id}')
```
Comment (dwppalmer95) [inline client/src/pages/UserProfile.jsx:13]: the string inside the fetch request needs a template literal (between backticks ``) to allow for variable referencing.

**Review by rmaiorca — APPROVED** [review]

File: server/models/DB.sql [hunk 2]
```diff
@@ -8,26 +8,36 @@ CREATE TABLE users (
...
-  full_name varchar (100) NOT NULL,
-  profile_pic VARCHAR DEFAULT 'userProfile.png',	
   overall_rating decimal,
   respect_rating decimal,
   responsiveness_rating decimal,
   bike_friendly boolean,
   pet_friendly boolean,
   is_verified boolean DEFAULT false,
-  created_at timestamp DEFAULT CURRENT_TIMESTAMP
+  FOREIGN KEY (_id) REFERENCES users(_id),
```
Comment (dwppalmer95) [inline server/models/DB.sql:23]: This should be `user_id INT NOT NULL FOREIGN KEY REFERENCE users(_id)`. Each table should have its own unique primary key (`_id`), so you'll want to add an additional column to the table specifically for the foreign key.

File: server/models/DB.sql [hunk 3]
```diff
@@ -8,26 +8,36 @@ CREATE TABLE users (
...
   email varchar (50) NOT NULL UNIQUE,
   password varchar (100) NOT NULL,
-  is_landlord boolean DEFAULT false,
-  landlord_id integer,
-  created_at timestamp DEFAULT CURRENT_TIMESTAMP
+  profile_pic VARCHAR DEFAULT 'userProfile.png',	
+  is_landlord boolean DEFAULT false NOT NULL,
 );
 
 CREATE TABLE landlords(
```
Comment (dwppalmer95) [inline server/models/DB.sql:15]: Add `user_id` as a column in the table. See comment below.

File: server/models/DB.sql (hunk 2 above)
Comment (rmaiorca) [reply server/models/DB.sql:23]: Does this need to be changed on the landlordAddresses table too?
Also - should it be user_id or users_id?

**Pull request merged**